pubsub/pubsub.go

```diff
@@ -159,15 +159,11 @@ type Subscription struct {
 	// ackBatcher makes batches of acks and sends them to the server.
 	ackBatcher driver.Batcher
 
-	mu sync.Mutex
-
-	// q is the local queue of messages downloaded from the server.
-	q     []*Message
-	err   error
-	waitc chan struct{} // for goroutines waiting on ReceiveBatch
-
-	// cancel cancels all SendAcks calls.
-	cancel func()
+	mu     sync.Mutex
```

Comment:
Shouldn't mu have a comment telling what it protects?

Reply:
By convention, it protects everything underneath it until the next blank line. I'll add that though.